src/actions/index.js

```diff
@@ -14,3 +15,13 @@ export function fetchPosts() {
     };
 
 }
+
+
+export function createPost(props) {
+    const request = axios.post(`${ROOT_URL}/posts${API_KEY}`, props);
```

Comment:
My understanding is that the axios has already fetched the response here. We should name the const `request` to `new_post`. Also have you considered to use `catch`  function here incase server throws error?

Reply:
Yes, you are right. I have not considered a server error thus far. I will address these changes.